lightning/src/routing/network_graph.rs

```diff
@@ -1394,7 +1394,11 @@ impl NetworkGraph {
 						} else {
 							chan_was_enabled = false;
 						}
+					}
+				}
 
+				macro_rules! maybe_update_channel_info {
+					( $target: expr, $src_node: expr) => {
```

Comment:
Looks like `$src_node` is not used in either macro. Probably could get rid of `$target`, too, and just have the macro evaluate to the `ChannelUpdateInfo` and assign from call sites.